packages/@react-spectrum/datepicker/test/DateField.test.js

```diff
@@ -167,14 +167,12 @@ describe('DateField', function () {
     });
 
     it('should support format help text', function () {
-      let {getByRole, getByText, getAllByRole} = render(<DateField label="Date" showFormatHelpText />);
+      let {getByRole, getAllByRole} = render(<DateField label="Date" showFormatHelpText />);
 
       // Not needed in aria-described by because each segment has a label already, so this would be duplicative.
       let group = getByRole('group');
       expect(group).not.toHaveAttribute('aria-describedby');
 
-      expect(getByText('month / day / year')).toBeVisible();
```

Comment:
i removed this line since now each individual segment is in its own span and it would be hard to query those. we have a chromatic story to test this so it is covered. i kept the rest of the test since it would be good to continue checking the aria stuff.